tool/dev.dart

```diff
@@ -82,24 +82,6 @@ List<String> _serverOutput;
 /// Output from the SockJS server.
 List<String> _sockJSServerOutput;
 
-/// Start the server needed for integration tests and examples and cache the
-/// server output until the task requiring the server has finished. Then, the
-/// server output will be dumped all at once.
-Future _startServer() async {
-  _serverOutput = [];
-  _server = new Server();
-  _server.output.listen(_serverOutput.add);
-  await _server.start();
-}
-
-Future _startSockJSServer() async {
-  _sockJSServerOutput = [];
-  _sockJSServer = new TaskProcess('node', ['tool/server/sockjs.js']);
-  _sockJSServer.stdout.listen(_sockJSServerOutput.add);
-  _sockJSServer.stderr.listen(_sockJSServerOutput.add);
-  // todo: wait for server to start
-}
```

Comment:
Buffering the output of the integration servers until the end of the test run was pretty useless - there's no way to map a log entry to the test the generated it, so I switched the test/coverage tasks to use the streaming option so that the logs are placed in line with the tests as they run.